src/editor/svgedit.js

```diff
@@ -371,11 +350,11 @@ function getImportLocale ({defaultLang, defaultName}) {
      * @param {string} language
      * @returns {Promise<module:locale.LocaleStrings>} Resolves to {@link module:locale.LocaleStrings}
      */
-    function importLocale (language) {
+    async function importLocale (language) {
       const url = `${curConfig.extPath}ext-locale/${name}/${language}.js`;
-      return importSetGlobalDefault(url, {
-        global: `svgEditorExtensionLocale_${name}_${language.replace(/-/g, '_')}`
-      });
+      // eslint-disable-next-line node/no-unsupported-features/es-syntax
+      const locale = await import(url);
```

Comment:
It's incomplete now but this should work. By the way, how do you test old browsers?

Other more generic question: many applications tends to drop support to old browsers and to suggest to use an older version of svgedit if they have no choice (svgedit 5.x for us)?

I think we are adding a lot of complexity to maintain svgedit because if this support for old browsers (which by the way cause security concerns).